openpype/hosts/blender/plugins/publish/collect_review.py

```diff
@@ -62,45 +62,10 @@ def process(self, instance):
             and obj.type in ("MESH", "CURVE", "SURFACE")
         ]
 
-        reviewable_instances = [
-            context_instance
-            for context_instance in instance.context
-            if context_instance.data.get("family")
-            not in ("review", "camera", "workfile")
-        ]
-
-        if reviewable_instances:
-            if len(reviewable_instances) > 1:
-                self.log.warning(
-                    f"Multiple subsets for review {reviewable_instances}"
-                )
-
-            reviewable_instance = reviewable_instances[0]
-            self.log.debug(f"Subset for review: {reviewable_instance}")
-
-            if not isinstance(reviewable_instance.data.get("families"), list):
-                reviewable_instance.data["families"] = []
-            reviewable_instance.data["families"].append("review")
-
-            reviewable_instance.data.update(
-                {
-                    "review_camera": camera,
-                    "frameStart": instance.context.data["frameStart"],
-                    "frameEnd": instance.context.data["frameEnd"],
-                    "fps": instance.context.data["fps"],
-                    "isolate": isolate_objects,
-                    "audio": audio_tracks,
-                }
-            )
-            instance.data["remove"] = True
-
         if not instance.data.get("remove"):
```

Comment:
You can remove this condition line and delete this file : `openpype\hosts\blender\plugins\publish\collect_remove_marked.py`
The OpenPype team cleaned up their Maya revision system and it should never be useful again.

LGTM otherwise.